src/main/java/mate/controller/IndexController.java

```diff
@@ -0,0 +1,17 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class IndexController extends HttpServlet {
+    private static final String INDEX_ADDRESS = "/WEB-INF/views/index.jsp";
```

Comment:
btw, in his case constant is not obligatory. We use it if string is repeated in code for easier editing or if string needs informative name to understand its' purpose